plugins/github/github.plugin.zsh

```diff
@@ -68,8 +68,9 @@ exist_gh() { # [DIRECTORY]
 # documentation: https://github.com/blog/985-git-io-github-url-shortener
 #
 git.io() {
-  emulate -L zsh
-  curl -i -s https://git.io -F "url=$1" | grep "Location" | cut -f 2 -d " "
+  # emulate -L zsh
```

Comment:
What do you think about deleting these lines instead of commenting them out?

Reply:
I just commented out the code for archive purposes.

This way of submitting links is never officially documented by GitHub, so even if `git.io` stop accepting new link I'd like to keep it as a reference.

Reply:
That is the purpose source control. If the comment doesn't have enough info maybe add more detail?

Reply:
> That is the purpose of source control. If the comment doesn't have enough info maybe add more detail?

Sounds fair.

> To be clear. I'm talking about the code lines you commented out. Keeping the git.io comment is good.

Yeah, I know that.